baybe/utils/validation.py

```diff
@@ -83,31 +84,32 @@ def validate_target_input(data: pd.DataFrame, targets: Iterable[Target]) -> None
         targets: The allowed targets.
 
     Raises:
-        ValueError: If the input dataframe is empty.
-        ValueError: If any target data contain NaN.
+        ValueError: If data is empty.
+        ValueError: If data misses columns for a target.
```

Comment:
If you insist using the variable reference (which I don't see the necessity for) then you must use backticks